content/en/docs/setup/install/multicluster/_index.md

```diff
@@ -22,7 +22,7 @@ This guide covers some of the most common concerns when creating a
 {{< gloss >}}multicluster{{< /gloss >}} mesh:
 
 - [Network topologies](/docs/ops/deployment/deployment-models#network-models):
-  one or two networks
+  one or more networks.
```

Comment:
If not being specific about the cases covered by the doc, then maybe you should also change line 29 for consistency:
```
primary and {{< gloss "remote cluster">}}remote clusters{{< /gloss >}}
```

Reply:
I've reverted this and added a tip below.